pkg/types/report.go

```diff
@@ -100,16 +100,17 @@ var (
 
 // Result holds a target and detected vulnerabilities
 type Result struct {
-	Target            string                     `json:"Target"`
-	Class             ResultClass                `json:"Class,omitempty"`
-	Type              ftypes.TargetType          `json:"Type,omitempty"`
-	Packages          []ftypes.Package           `json:"Packages,omitempty"`
-	Vulnerabilities   []DetectedVulnerability    `json:"Vulnerabilities,omitempty"`
-	MisconfSummary    *MisconfSummary            `json:"MisconfSummary,omitempty"`
-	Misconfigurations []DetectedMisconfiguration `json:"Misconfigurations,omitempty"`
-	Secrets           []ftypes.SecretFinding     `json:"Secrets,omitempty"`
-	Licenses          []DetectedLicense          `json:"Licenses,omitempty"`
-	CustomResources   []ftypes.CustomResource    `json:"CustomResources,omitempty"`
+	Target                 string                     `json:"Target"`
+	Class                  ResultClass                `json:"Class,omitempty"`
+	Type                   ftypes.TargetType          `json:"Type,omitempty"`
+	Packages               []ftypes.Package           `json:"Packages,omitempty"`
+	Vulnerabilities        []DetectedVulnerability    `json:"Vulnerabilities,omitempty"`
+	IgnoredVulnerabilities []DetectedVulnerability    `json:"IgnoredVulnerabilities,omitempty"`
```

Comment:
Trivy can ignore misconfigurations and other security issues. We need to carefully consider the implementation.

In addition, after VEX, vulnerabilities can be suppressed for some reason. I was considering how to manage it and didn't come up with a good idea.

Reply:
1) I feel like future issues/PRs could be created for misconfigurations/et al. Showing the ignored vulnerabilities seems like a good start, this is my opinion.

 2) Are you implying that VEX is incorrectly suppressing some vulnerabilities or is this a desired behavior? Not sure I understand what you mean.